src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4346,7 +4362,26 @@ public override bool PreVisit(CallNode node)
                 _currentScope = scopeNew.Parent;
                 PostVisit(node.Args);
 
-                // Typecheck the invocation.
+                if (maybeFunc.HasIdentifiers && _features.HasFlag(Features.SupportColumnNamesAsIdentifiers))
+                {
+                    var i = 0;
+
+                    foreach (var arg in args)
+                    {
+                        i++;
+
+                        if (arg is FirstNameNode firstNameNode && maybeFunc.IsIdentifierParam(i))
+                        {
+                            var displayName = GetLogicalNodeNameAndUpdateDisplayNames(argTypes[0], firstNameNode.Ident, out _);
+
+                            if (argTypes[0].TryGetType(displayName, out var fieldType))
+                            {
+                                var fileNameInfo = FirstNameInfo.Create(firstNameNode, new NameLookupInfo(BindKind.NamedValue, fieldType, DPath.Root, 0, displayName: new DName(displayName)));
+                                _txb.SetInfo(firstNameNode, fileNameInfo);
```

Comment:
>_txb.SetInfo(firstNameNode, fileNameInfo); [](http://example.com/codeflow?start=32&length=42)

Is there anything that actually uses this FirstNameInfo that we set here?

Reply:
After checking ... no!
I'll remove that part